config/README.md

```diff
@@ -0,0 +1,146 @@
+# Configuration
```

Comment:
I believe all environment variable should have `HEZNODE_` prefix to avoid collision with other applications that might have in the server.